### PR TITLE
feat(css/minifier): compress import at-rule

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Compress `@import` at-rule

**BREAKING CHANGE:**

No

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_minifier/src/lib.rs [hunk 1]
```diff
@@ -28,5 +28,5 @@ pub fn minify(stylesheet: &mut Stylesheet) {
     stylesheet.visit_mut_with(&mut compress_declaration());
     stylesheet.visit_mut_with(&mut compress_selector());
     stylesheet.visit_mut_with(&mut compress_keyframes());
-    stylesheet.visit_mut_with(&mut compress_selector());
+    stylesheet.visit_mut_with(&mut compress_at_rule());
```
Comment (alexander-akait) [inline crates/swc_css_minifier/src/lib.rs:31]: Looks like we do mistake when merge and we have two `compress_selector`

**Review by kdy1 — APPROVED** [review]
LGTM. Thanks!

---

swc-bump:
 - swc_css_minifier

**Review by kdy1 — CHANGES_REQUESTED** [review]
Clippy is angry again

**Comment by alexander-akait** [comment]
Close in favor https://github.com/swc-project/swc/pull/3850

**Pull request closed without merging**